docs/features/comments.md

```diff
@@ -63,6 +63,18 @@ type Options = {
     category: string
     categoryId: string
 
+    // Url to folder with custom themes
+    // defaults to 'https://giscus.app/themes'
+    themeUrl?: string
```

Comment:
I think we should default this to baseUrl instead of allowing arbitrary URL here

Reply:
Then we can add a base configuration for quartz fwiw

Reply:
I have added a basic Quartz theme for both dark and light mode, as well as using this theme by default.